app/routes/_index.tsx

```diff
@@ -35,19 +53,31 @@ export default function Index() {
             typografi={TypografiTyper.StegHeadingH1}
           />
           <Språkvelger />
+
           <VeilederHilsen
             innhold={tekster.veilederhilsenInnhold}
             hilsen={tekster.brukerHilsen}
           />
           <SamtykkePanel
+            tittel={tekster.samtykkePanelTittel}
             innhold={tekster.samtykkePanelMelding}
             samtykke={tekster.samtykkePanelSamtykke}
             feilmelding={tekster.samtykkePanelFeilmelding}
+            onSamtykkeEndring={håndtereSamtykkeEndring}
+            feilmeldingAktivert={feilmeldingAktivert}
           />
+          <Button
+            variant={samtykkeErBekreftet ? 'primary' : 'secondary'}
```

Comment:
```suggestion
            variant={samtykkeErBekreftet ? 'primary' : 'secondary'}
```

Var det slik at "Gå videre" knappen skulle endre variant underveis?

Reply:
Ja, dette var oppklaringen:

- Det er riktig at knappen skal bli blå når valideringen er riktig.
- Da må man validere underveis, men uten å vise feilmelding
- Dersom man har prøvd å gå videre en gang, da skal feilmeldingene vises
- Feilmeldingen kan oppdateres underveis som man skriver
- Jeg tenker at tomt felt kan være: "Du må fylle inn tekst". minst 1 men mindre 10 kan være "Du må fylle inn minimum 10 tegn"